types/cote/index.d.ts

```diff
@@ -115,6 +115,11 @@ export interface ResponderAdvertisement extends Advertisement {
      * Request types that a Responder can listen to.
      */
     respondsTo?: string[];
+
+    /**
+     * Subset attribut for directed requests.
```

Comment:
Worth changing to something like "Advertisement attribute that lets you target a subgroup of responders using the __subset property of a request." because from upstream it's not immediately obvious how to use this. Can you please update?

Reply:
Effectively your comment is better than mine, I update it